content/workshops/openshift33/exercise-140-rollbacks.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Webhooks and Rollbacks
+workshops: openshift33
+workshop_weight: 140
+layout: lab
+---
+
+### Build Triggers, Webhooks and Rollbacks - Oh My!
+Once you have an app deployed in OpenShift you can take advantage of some continuous capabilities that help to enable DevOps and automate your management process.  We will cover some of those in this lab: Build triggers, webhooks, and rollbacks.
+
+
+### A bit of configuration
+We are going to do some integration and coding with an external git repository.  For this lab we are going to use github, if you don't already have an account, [you can create one here][3].
+
+OK, let's fork the dc-metro-map app from **my** account into **your** github account.  Goto [https://github.com/dudash/openshift-workshops/][4] and look to the top right for the "Fork" button.
```

Comment:
I forked this into RedHatGov so that it looks more official. It needs to be cleaned up though, I would like it to be versioned for 3.3 and later versions as well, and only contain source code for the lab and delete the documentation in that repo. 

https://github.com/RedHatGov/openshift-workshops